### ISSUE TITLE
A couple of related deprecations

### ISSUE THREAD

**Issue by ehymel** [secit-pl/imap-bundle#31]
Using symfony 6.2 and version 2.0.0 of this bundle (secit-pl/imap-bundle), I am getting these deprecations:

```
The "Symfony\Component\Console\Command\Command::$defaultName" property is considered final. You should not override it in "SecIT\ImapBundle\Command\ValidateConnectionsCommand".

The "Symfony\Component\Console\Command\Command::$defaultDescription" property is considered final. You should not override it in "SecIT\ImapBundle\Command\ValidateConnectionsCommand".

Since symfony/console 6.1: Relying on the static property "$defaultName" for setting a command name is deprecated. Add the "Symfony\Component\Console\Attribute\AsCommand" attribute to the "SecIT\ImapBundle\Command\ValidateConnectionsCommand" class instead.

Since symfony/console 6.1: Relying on the static property "$defaultDescription" for setting a command description is deprecated. Add the "Symfony\Component\Console\Attribute\AsCommand" attribute to the "SecIT\ImapBundle\Command\ValidateConnectionsCommand" class instead.

```

Thank you

**Comment by ehymel** [comment]
*bump*

**Comment by russelomua** [comment]
up

**Comment by secit-pl** [comment]
Sorry for the lack of response. I have been very busy the last few months. 

I'm working on the new version of this bundle with a lot of improvements, it will be released as 3.0.0 about next week.
